thrust/thrust/version.h

```diff
@@ -49,13 +51,15 @@ _CCCL_IMPLICIT_SYSTEM_HEADER
 
 /*! \def THRUST_VERSION
  *  \brief The preprocessor macro \p THRUST_VERSION encodes the version
- *         number of the Thrust library.
+ *         number of the Thrust library as MMMmmmpp.
+ *
+ *  \note THRUST_VERSION is formatted as `MMMmmmpp`, which differs from `CCCL_VERSION` that uses `MMMmmmppp`.
  *
  *         <tt>THRUST_VERSION % 100</tt> is the sub-minor version.
  *         <tt>THRUST_VERSION / 100 % 1000</tt> is the minor version.
  *         <tt>THRUST_VERSION / 100000</tt> is the major version.
  */
-#define THRUST_VERSION 200200
+#define THRUST_VERSION 200300 // macro expansion with ## requires this to be a single value
```

Comment:
Can you give an example of where code is breaking is it our ABI namespace thing?

Also I would prefer if we could then at least update the other defines below

Reply:
https://godbolt.org/z/a1bGccr4W